### PR TITLE
Add doc for new `contains` jinja filter and test

### PR DESCRIPTION
**Pull request by Vaarlion**
## Proposed change

Add some doc for the new jinja test and filter `contains` proposed in https://github.com/home-assistant/core/pull/86390

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/86390
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
This PR is targeting an incorrect branch. As it has a parent PR in the core codebase, it depends on a next release of Home Assistant and thus needs to go into the `next` branch.

Please change the target PR of this PR to the `next` branch and git rebase your changes onto the upstream `next` branch to ensure your PR is clean without unintended PRs coming along.

../Frenck

**Comment by Vaarlion** [comment]
My apologist for the time lost having to tell me what i should have known.
This is now rebased and changed.

File: source/_docs/configuration/templating.markdown [hunk 1]
```diff
@@ -696,16 +696,42 @@ It will also work as a filter over an iterable group of entities or groups:
...
+Closest to some entity:
     {{ ['group.children', states.device_tracker] | closest(states.zone.school) }}
 ```
 
 {% endraw %}
 
+### Contains
+
+Jinja provide by default a [`in` operator](https://jinja.palletsprojects.com/en/latest/templates/#other-operators) how return `True` when one element is `in` a provided list.
+The `contains` test and filter allow you to do the exact opposite and test for a list containing an element. This is particularly usefully in `select` or `selectattr` filter, as well as to check a if a device have a specific attribute, a `supported_color_modes`, a specific light effect...
```
Comment (frenck) [inline source/_docs/configuration/templating.markdown]:
```suggestion
Jinja provides by default a [`in` operator](https://jinja.palletsprojects.com/en/latest/templates/#other-operators) how return `True` when one element is `in` a provided list.
The `contains` test and filter allow you to do the exact opposite and test for a list containing an element. This is particularly useful in `select` or `selectattr` filter, as well as to check if a device has a specific attribute, a `supported_color_modes`, a specific light effect.
```

File: source/_docs/configuration/templating.markdown [hunk 2]
```diff
@@ -696,16 +696,42 @@ It will also work as a filter over an iterable group of entities or groups:
...
+
+Some examples:
+{% raw %}
+
+- `{{ state_attr('light.dining_room', 'effect_list') | contains('rainbow') }}` will return `true` if the light have a `rainbow` effect
```
Comment (frenck) [inline source/_docs/configuration/templating.markdown]:
```suggestion
- `{{ state_attr('light.dining_room', 'effect_list') | contains('rainbow') }}` will return `true` if the light has a `rainbow` effect.
```

File: source/_docs/configuration/templating.markdown [hunk 3]
```diff
@@ -696,16 +696,42 @@ It will also work as a filter over an iterable group of entities or groups:
...
+    {% set current_month = now().month %}
+    {% set extra_ambiance = [
+      {'name':'Halloween', 'month': [10,11]},
+      {'name':'Noel', 'month': [1,11,12]}
+    ]%}
+    {% set to_add = extra_ambiance | selectattr('month', 'contains', current_month ) | map(attribute='name') | list  %}
+    {% set to_remove = extra_ambiance | map(attribute='name') | reject('in', to_add) | list %}
+    {{ (state_attr('input_select.light_theme', 'options') + to_add ) | unique | reject('in', to_remove) | list }}
+  ```
+  This more complex example use the `contains` filter to match the current month with a list. In this case it's used to generate a list of light theme to give to the `Input select: Set options` service.
```
Comment (frenck) [inline source/_docs/configuration/templating.markdown]:
```suggestion
  This more complex example uses the `contains` filter to match the current month with a list. In this case, it's used to generate a list of light theme to give to the `Input select: Set options` service.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/configuration/templating.markdown (hunk 3 above)
Comment (Vaarlion) [reply source/_docs/configuration/templating.markdown]: Fixed thank

File: source/_docs/configuration/templating.markdown (hunk 2 above)
Comment (Vaarlion) [reply source/_docs/configuration/templating.markdown]: Fixed thank

File: source/_docs/configuration/templating.markdown (hunk 1 above)
Comment (Vaarlion) [reply source/_docs/configuration/templating.markdown]: Fixed thank

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**